palace/utils/timer.hpp

```diff
@@ -22,104 +24,171 @@ class Timer
   {
     INIT = 0,
     CONSTRUCT,
+    FSS,
```

Comment:
`FREQSPACESAMPLING`?

Reply:
Good to spell it out, though do you prefer to call it that or `FREQSAMPLESELECTION` to mirror how its printed now?

Reply:
That works too, I was just trying to save a few characters. Whatever you prefer!